src/applications/facility-locator/containers/FacilityDetail.jsx

```diff
@@ -111,7 +110,9 @@ class FacilityDetail extends Component {
     const isVBA = facilityType === FacilityType.VA_BENEFITS_FACILITY;
     return (
       <div>
-        <h1>{name}</h1>
+        <h1 ref={this.headerRef} tabIndex={0}>
```

Comment:
should `tabIndex` be set to -1 so we can set focus to it, but the user cannot tab to it as it is not an interactive element?

Reply:
Yes, tabIndex should be set to -1!

Reply:
Yes, thank you Ryan. I'll make that change.

Reply:
Learn something new with this tabIndex attribute!